qemu/qemu.go

```diff
@@ -288,6 +288,22 @@ func (object Object) QemuParams(config *Config) []string {
 	return qemuParams
 }
 
+// Virtio9PMultidev filesystem behaviour to deal
+// with multiple devices being shared with a 9p export.
+type Virtio9PMultidev string
+
+const (
+	// Remap shares multiple devices with only one export.
+	Remap Virtio9PMultidev = "remap"
+
+	// Warn assumes that only one device is shared by the same export.
+	// Only a warning message is logged (once) by qemu on host side.
+	Warn Virtio9PMultidev = "warn"
```

Comment:
Where is this used?

Reply:
this is the default behaviour

Reply:
As the original documentation, on QEMU side, I'd mention this one is the original behaviour.

Reply:
fixed, thanks